components/site/tests/site.rs

```diff
@@ -629,3 +629,13 @@ fn can_apply_page_templates() {
     assert_eq!(child.meta.template, Some("page_template_child.html".into()));
     assert_eq!(child.meta.title, Some("Local section override".into()));
 }
+
+// https://github.com/getzola/zola/issues/571
+#[test]
+fn can_build_site_with_404_from_theme() {
```

Comment:
can you remove the 404 from the title? Something like `can_build_site_custom_builtins_templates_from_theme`

Reply:
Function changed with the appropriate name.